core/src/test/scala/integration/kafka/api/PlaintextProducerSendTest.scala

```diff
@@ -121,16 +122,18 @@ class PlaintextProducerSendTest extends BaseProducerSendTest {
   }
 
   @ParameterizedTest(name = TestInfoUtils.TestWithParameterizedQuorumName)
-  @ValueSource(strings = Array("zk", "kraft"))
-  def testSendWithInvalidCreateTime(quorum: String): Unit = {
+  @MethodSource(Array("quorumAndTimestampConfigProvider"))
```

Comment:
@mehbey this test is failing in CI with errors and it looks related to this change. Please look into it when you get a chance.


https://ci-builds.apache.org/job/Kafka/job/kafka-pr/job/PR-14135/9/testReport/junit/kafka.api/PlaintextProducerSendTest/Build___JDK_17_and_Scala_2_13___initializationError/

Reply:
That was an oversight from my side when changing the parameter provider to include `quorum`. Updated and verified that all tests are passing in this file. Will check back again to verify that CI is happy.